using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -146,6 +146,30 @@ Here are the key-value pairs in this event:
 }
 ```
 
+### Generic Data OS <a id="generic-data-os"></a>
+
+This event is dispatched by the Core extension to the EventHub when the collectMessageInfo API is invoked by the app developer to track local and push notification interactions.
+
+#### Data payload definition <a id="data-payload-definition-7"></a>
+
+Here are the key-value pairs in this event:
+
+| **Key**      | **Value Type** | **Optional** | **Description**                                              |
+| :----------- | :------------- | :----------- | :----------------------------------------------------------- |
+| `deliveryId` | String         | No           | String containing the delivery id of the message interacted with. |
+| `action`     | String         | No           | String containing the message interaction id.                |
+| `broadlogId` | String         | No           | String containing the broadlog id of the message interacted with. |
```

Comment:
We can just say `String containing the broadlog ID for the message` or `String containing the delivery ID for the message`.